Public/Src/Engine/Scheduler/Fingerprints/PipFingerprinter.cs

```diff
@@ -248,7 +250,19 @@ protected virtual void AddWeakFingerprint(IFingerprinter fingerprinter, SealDire
         /// </summary>
         protected virtual void AddWeakFingerprint(IFingerprinter fingerprinter, Process process)
         {
-            AddFileDependency(fingerprinter, "Executable", process.Executable);
+            bool untrackedExecutable = false;

```

Comment:
Please don't reassign value to variable whenever possible
```suggestion
            bool untrackedExecutable = 
                process.UntrackedPaths.Contains(process.Executable.Path) ||
                process.UntrackedScopes.Any(scope => process.Executable.Path.IsWithin(m_pathTable, scope));
```